dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -203,22 +203,10 @@ def authenticate
         user = Services::Lti.initialize_lti_user(decoded_jwt)
         # PartialRegistration removes the email address, so store it in a local variable first
         email_address = Services::Lti.get_claim(decoded_jwt, :email)
+        Services::Lti.initialize_lms_landing_session(session, integration[:platform_name], 'new', user.user_type)
         PartialRegistration.persist_attributes(session, user)
-        if DCDO.get('lti_account_linking_enabled', false)
-          Services::Lti.initialize_lms_landing_session(session, integration[:platform_name], 'new', user.user_type)
-          publish_linking_page_visit(user, integration[:platform_name])
-          render 'lti/v1/account_linking/landing', locals: {email: email_address} and return
-        end
-
-        if DCDO.get('student-email-post-enabled', false)
```

Comment:
@stephenliang this removes a block of code that checks for the student email post flag. Don't think it'll affect anything in prod though, since this code is unreachable if the account linking flag is enabled.